apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -27,6 +28,16 @@ namespace Google.Cloud.Diagnostics.AspNet
     /// </summary>
     internal static class Labels
     {
+        /// <summary>The version of the trace agent.</summary>
+        internal static readonly string AgentVersion = typeof(Labels)
+            .GetTypeInfo()
+            .Assembly
+            .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
+            .InformationalVersion;
```

Comment:
@jskeet Do we need to worry about the library not having reflection permissions here? Should this be moved to a helper where it can gracefully degrade when lacking permissions.